README.md

```diff
@@ -4,4 +4,99 @@ This repository contains plugins for
 [Yosys](https://github.com/YosysHQ/yosys.git) developed as
 [part of the SymbiFlow project](https://symbiflow.github.io).
 
+## List of plugins
+1. [Design introspection](#design_introspection)
+2. [Fasm](#fasm)
+3. [Get count](#get_count)
+4. [Integrate inverters](#integrate_inv)
+5. [Parameters](#parameters) 
+6. [QL IOBs](#ql_iob)
+7. [SDC](#sdc)
+8. [Selection](#selection)
+9. [XDC](#xdc)
 
+## Summary
+
+### Design introspection plugin <a name="design_introspection"></a>
```

Comment:
I don't think you need these `<a name="design_introspection">` as the headers should already have that.

Reply:
Fixed